### DISCUSSION TITLE
how to cancel jump function when I use "/" or "?" to search current file

### DISCUSSION THREAD

**Discussion by coderup2021** [LazyVim/LazyVim#1630] [Q&A] [answered]
When I use "/" to search current file, it will show some extra letter for jump, I don't want this jump function, the extra letter often conflict with my search letter, so i want to cancel this function.

Anyone knows? please help!

![企业微信截图_8d85a36f-e649-47f8-8d25-cf69ba01b736](https://github.com/LazyVim/LazyVim/assets/93026018/4e4b5927-6812-4c19-a8ab-123c4612e830)
![企业微信截图_63cad0db-827b-468a-be1d-d72eed3a91cb](https://github.com/LazyVim/LazyVim/assets/93026018/e0c952db-d7ac-4425-b6d5-96aca8e4e5cd)

**Comment by folke** [comment]
You can cancel the jumps by doing `<C-S>` while searching.
To disable it fully, set `opts.mode.search.enabled = false` for the flash.nvim plugin

**Reply by timjackleus** [reply]
Thank you I've been wondering why this is happening as well

**Reply by coderup2021** [reply, marked as answer]
thank you very much, have tried `<C-S>` while searching, it's working.

but to disable it fully, it is `opts.modes.search.enabled = false`.

I touch a file: ~/.config/nvim/lua/plugins/flash-nvim.lua  with the content:
```
return {
  "folke/flash.nvim",
  opts = {
    modes = {
      search = {
        enabled = false,
      },
    },
  },
}
```
it works.

**Comment by coderup2021** [comment]
> Thank you I've been wondering why this is happening as well

For Example:
with following content:
```
MIB_TBL_ENTRY aaaaaaaaaaaaaaaaaaaaaaa
MIB_TBL_ENTRY bbbbbbbbbbbbbbbbbbbbbbbbbbb
MIB_TBL_ENTRY ccccccccccccccccccccccccc
MIB_TBL_ENTRY ddddddddddddddddddddddd
MIB_TBL_ENTRY eeeeeeeeeeeeeeeeeeeeeeeee
MIB_TBL_ENTRY fffffffffffffffffffffffffff
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
MIB_TBL_ENTRY gggggggggggggggggggggggggggg
```
When I search `TBL`, I can not input the word completely, it will jump to some place, I guess if the search target is more repeated, it have more chance to conflict
